PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalAnalytics.kt

```diff
@@ -7,4 +7,12 @@ internal object PayPalAnalytics {
     const val TOKENIZATION_FAILED = "paypal:tokenize:failed"
     const val TOKENIZATION_SUCCEEDED = "paypal:tokenize:succeeded"
     const val BROWSER_LOGIN_CANCELED = "paypal:tokenize:browser-login:canceled"
+
+    const val EDITFI_STARTED = "paypal:edit:started"
+    const val EDITFI_BROWSER_PRESENTATION_SUCCEEDED = "paypal:edit:browser-presentation:succeeded"
+    const val EDITFI_BROWSER_PRESENTATION_FAILED = "paypal:edit:browser-presentation:failed"
+    const val EDITFI_BROWSER_LOGIN_ALERT_CANCELED = "paypal:edit:browser-login:alert-canceled"
```

Comment:
```suggestion
```
`browser-login:alert-canceled` is iOS specific according to the FPTI events excel sheet, so can be removed for Android